pages/index.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import SomeComponent from 'client/components/SomeComponent';
+
+export default () => {
+  return (
+    <div>
+      this is a page!
+      <SomeComponent />
+      <style jsx>{`
```

Comment:
Wow, I haven't seen this kind of styling before. Is there a name to it that I can possibly Google and learn more about? TIA!

Reply:
Yup! `styled jsx` is the name; it comes shipped with next.js